org.gradle.toolingapi/META-INF/MANIFEST.MF

```diff
@@ -9,8 +9,6 @@ Require-Bundle: org.slf4j.api;bundle-version="1.7.2"
 Export-Package: org.gradle;version="4.3.0",
```

Comment:
Is org.gradle really part of the tooling API? It might just be here because it is used by the TAPI internally. I don't remember any classes from that package that you would need to use ProjectConnection. We should hide as much as possible.

Reply:
I've already found one exception: `import org.gradle.api.JavaVersion` is used in the tooling models.

Reply:
I managed to remove a bunch of unused entries from the exported packages list. Only the following ones remained:
- `org.gradle.tooling.*`: obviously
- `org.gradle.tooling`: because of `JavaVersion`
- `org.gradle.util`: because of `GradleVersion`